src/main/java/io/jsonwebtoken/impl/ValidationHelper.java

```diff
@@ -0,0 +1,193 @@
+package io.jsonwebtoken.impl;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Clock;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Header;
+import io.jsonwebtoken.PrematureJwtException;
+
+/**
+ * This class contains static methods for validation of JWT claims.
+ */
+public class ValidationHelper {
```

Comment:
Please no 'Helper' or 'Utils' or such classes for anything other than things that augment the Java language itself (e.g. Strings.whatever, Objects.whatever, Assert.whatever are fine because they augment the Java language and those things are often not overridable - such as String being final) - these typically often represent procedural (non OO) logic.

If anything, this would be a `Validator` or something similarly named.  Additionally, anything that can be configured/overridable should be an interface and a default implementation provided so that people can plug in different implementations or override them as desired, e.g. `Validator` and `DefaultValidator` (assuming it is correct to have such a component).